### ISSUE TITLE
Plurals work for `en` but not for `ro`

### ISSUE THREAD

**Issue by andreialecu** [i18next/i18next#1579]
## 🐛 Bug Report

It seems that pluralization is not working for Romanian. Not sure what I'm doing wrong here. See reproduction.

If I swap `ro` to `en` it works

## To Reproduce

https://jsfiddle.net/28zr4oLu/

## Expected behavior

Pluralization should work, the `addedFromPlaylist_plural` key should be used.

## Your Environment

- *i18next version*: 19.9.2

**Comment by jamuhl** [comment]
Romanian has more than one plural form: https://www.i18next.com/translation-function/plurals#languages-with-multiple-plurals

**Comment by andreialecu** [comment]
I'm a native Romanian speaker and I can't think of what the other plural form is. 🤔 

If I change the fiddle to this:

```
	  ro: {
        translation: {
          addedFromPlaylist_1: '**{{count}} song** from {{- playlistName}} {{duration}} has been added to the queue!',
          addedFromPlaylist: '**{{count}} songs** from {{- playlistName}} {{duration}} have been added to the queue!'
        }
      }
```

It displays:

```
**1 songs** from myPlaylist 1h have been added to the queue!
**2 song** from myPlaylist 1h has been added to the queue!
```

Which is the inverse of what the documentation you linked shows.

**Comment by jamuhl** [comment]
check https://www.i18next.com/translation-function/plurals#how-to-find-the-correct-plural-suffix

Language
ro -> Romanian, Moldavian, Moldovan (native: română)

Suffixes
key_0, key_1, key_2

**Comment by andreialecu** [comment]
Seems this has been reported before:

https://github.com/i18next/i18next/issues/1209#issuecomment-586027582

There was another Romanian guy there just as confused as me :)

The link you provided no longer works. Do you have any resources that would describe what the third plural is used for?

**Comment by andreialecu** [comment]
Alright, I was able to find what the third plural form is about. 

It's an extremely minor thing.

This link describes it (in Romanian):
https://localize.drupal.org/node/766

Leaving it here for future readers that may stumble upon this:

> nplurals=3: Limba română are 3 forme de plural. De la 0 la 2.
plural=...: Pluralul va folosi forma...
n==1 ? 0 : ...: Dacă avem un obiect folosim forma "0". Adică singularul: Un măr
n==1 ? ...  : (n==0 || (n%100 > 0 && n%100 < 20)) ? 1 : 2;: Dacă avem zero sau mai multe obiecte avem 2 forme:
Pentru numere din seriile: 0, 2-19, 101-119, 201-219, 301-219, ... forma "1": Zero mere, două mere, nouăsprezece mere, două sute unu mere.
Pentru nume din seriile: 20-100, 120-200, 220-300, 320-400, ... forma "2": Douăzeci de mere, o sută patruzeci de mere, trei sute douăzeci de mere.

Basically for ranges 20-100, 120-200, 220-300 there's an added preposition in some cases.

**Comment by andreialecu** [comment]
Question: Would it be possible for Romanian to support both the regular `key` and `key_plural`?

Having to define all 3 forms can be extremely error prone. For example:

https://jsfiddle.net/ucfnkgbs/

Notice how the fourth translation is broken.

As per https://github.com/i18next/i18next/issues/1209#issuecomment-586219091
> Ok, I got it, but in Romanian it only needs the 3rd form if you include the count number too, otherwise it is just as in English (singular or plural).I guess there is no easy way to define the plural if you know you will never add the count. In my case I will just duplicate `word_0` and `word_2`

I'd also not ever really need to differentiate the third form.

**Comment by jamuhl** [comment]
rule can be overridden https://github.com/i18next/i18next/blob/master/src/PluralResolver.js#L90

**Comment by andreialecu** [comment]
Alright. Thanks.

**Comment by gabrielgrijincu** [comment]
@jamuhl 
How can we use this `PluralResolver` to override rules? I tried to find some documentation about this, but nothing came up.
Thanks!

**Comment by jamuhl** [comment]
@gabrielgrijincu if i remember right it's something like `i18next.services.pluralResolver.addRule(lng, { plurals: function(n) {return Number(n > 1);} })`

**Comment by gabrielgrijincu** [comment]
Thank you very much, @jamuhl!
